tensorflow/core/util/mkl_heuristics.h

```diff
@@ -120,6 +120,73 @@ static double CalculateNodeMFlops(const AttrSlice& attrs,
   return -1;
 }
 
+// MatMulHeuristic returns true to rewrite the node with oneDNN
+// false to execute the node in Eigen
+static bool MatMulHeuristic(const Node* n) {
```

Comment:
(For future reference. I have already done this internally.)
Please `#include "tensorflow/core/graph/graph.h"` (below the `#include "tensorflow/core/framework/node_def_util.h"` line) for `tensorflow::Node`.

Reply:
Hi Penporn,

Does these header files fix the above three issues or do I have to look into these failures?

Reply:
Hi Crefeda, yes, this fixes a separate issue (can't find the definition for `tensorflow::Node`).